tests/safeds/data/tabular/typing/test_schema.py

```diff
@@ -33,24 +34,49 @@ class TestFromPandasDataFrame:
             ),
             (
                 pd.DataFrame({"A": [1, 2.0, "a", True]}),
-                Schema({"A": String()}),
+                Schema({"A": Anything()}),
             ),
             (
                 pd.DataFrame({"A": [1, 2, 3], "B": ["a", "b", "c"]}),
                 Schema({"A": Integer(), "B": String()}),
             ),
+            (
+                pd.DataFrame({"A": [True, False, None]}),
+                Schema({"A": Boolean(is_nullable=True)}),
+            ),
+            (
+                pd.DataFrame({"A": [1, None, 3]}),
+                Schema({"A": RealNumber()}),
```

Comment:
```suggestion
                Schema({"A": RealNumber(is_nullable=True)}),
```

Reply:
See https://github.com/Safe-DS/Stdlib/pull/432#pullrequestreview-1525308460 for explanation of this suggestion